web/v2/backend/src/main/java/com/tourneygen/web/Services/EloService.java

```diff
@@ -1,3 +1,6 @@
 package com.tourneygen.web.Services;
 
-public class EloService {}
+public class EloService {
```

Comment:
ELO is supposed to be a non-decimal number (at least in all uses of elo that I've seen.)

Would it constitute a lot of effort to make this an int?

Reply:
no id be pretty easy.But we could also just change the front end display

Reply:
Having the extra precision in the backend doesn't help anything, so I think the play here is to minimize the data rather than minimize the display in this case since it wouldn't be too much work

Reply:
eol is now int